examples/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
+# You can control the set of applied configuration files using the
+# proguardFiles setting in build.gradle.
+#
+# For more details, see
+#   http://developer.android.com/guide/developing/tools/proguard.html
+
+# If your project uses WebView with JS, uncomment the following
+# and specify the fully qualified class name to the JavaScript interface
+# class:
+#-keepclassmembers class fqcn.of.javascript.interface.for.webview {
+#   public *;
+#}
+
```

Comment:
Could you add additional ProGuard rules related to the usage of the classes `io.ably.lib.objects.DefaultObjectsPlugin` and `io.ably.lib.objects.serialization.DefaultObjectsSerializer`, as referenced in [ObjectsHelper.java](https://github.com/ably/ably-java/blob/main/lib/src/main/java/io/ably/lib/objects/ObjectsHelper.java), particularly for when obfuscation is enabled?

These rules will be necessary when building the release version of the code with obfuscation enabled.

We already have a [android/proguard.txt](https://github.com/ably/ably-java/blob/main/android/proguard.txt) file, so we can update that with the new rules.